firmware/tools/generate_custom_splash.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
Is this code a clone of `generate_world_map.bin.py`?

Reply:
Actually it is.
But generate_world_map.py does not accept arguments, and only convert a certain image.